test/firmata.test.js

```diff
@@ -515,6 +515,51 @@ describe("board", function() {
     serialPort.emit("data", [ANALOG_MESSAGE | (1 & 0xF), 0 % 128, 0 >> 7]);
   });
 
+  it("should be able to read value of analog pin on a board that skipped capabilities check", function(done) {
+    var serialPort = new SerialPort("/path/to/fake/usb");
+    var board = new Board(serialPort, {skipCapabilities: true, analogPins: [14,15,16,17,18,19]}, function(err) {});
```

Comment:
yeah, the test would definitely fail.  In fact the first commit on this PR is just the tests which are failing without any pin initializing.
I think any analog reading without some initializing might be a bigger refactor and likely a separate issue.